docs/en/commands/mobile-command.md

```diff
@@ -104,6 +104,7 @@ List of available commands:
 | mobile:activeAppInfo | Gets current active application's information such as bundleId and process arguments | <none> | <none> |
 | mobile:deviceInfo | Gets device information like locale and timezone. The timezone is TZ database format like `America/New_York` by default, fallback to Apple name like `US/New_York` | <none> | <none> |
 | mobile:rotateElement | Sends a rotation gesture with two touches to the given element. See [rotate:withVelocity:](https://developer.apple.com/documentation/xctest/xcuielement/1618665-rotate) for more details.  | `{element, rotation, velocity}` | `{element: 2, rotation: π/2, velocity: 1.5}` |
+| mobile:source | This endpoint allows to retrieve the current native page source from WDA as a string (also being in a web context). Possible output formats are: `xml` (same output as for `getPageSource` command, the default value), `json` (the same tree structure as for the `xml` format, but represented as JSON) and `description` (native `debugDescription` output generated by XCTest). | `{format}` | `{format: 'description'}` |
```

Comment:
You should update https://github.com/appium/appium/blob/master/commands-yml/commands/mobile-command.yml instead of generated `.md` :)

How about adding `excludedAttributes` like below?
```
`excludedAttributes` allows to exclude attributes  in the source. The format is like `excludedAttributes: ['visible']`. It helps to improve performance, especially excluding the visibility.
```

Reply:
all done